go/client/wallet_api_listen_display.go

```diff
@@ -47,32 +48,53 @@ func newWalletNotificationDisplay(g *libkb.GlobalContext) *walletNotificationDis
 		baseNotificationDisplay: newBaseNotificationDisplay(g),
 		cli:                     cli,
 		deduper:                 make(map[string]bool),
+		useV1:                   useV1,
 	}
 }
 
-func deduperKey(details stellar1.PaymentDetailsLocal) string {
```

Comment:
you could also leave this as deduperKey that takes an empty interface, switch on its type, and blow up if it gets something other than the two expected types.

Reply:
I kind of prefer getting compiler error about the type vs a runtime error. Ideally, we could define a type union or overload `deduperKey`...